src/tests/search.spec.js

```diff
@@ -0,0 +1,93 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import { describe, it } from 'mocha';
+import app from '../index';
+import helpers from '../utils/helper';
+
+chai.use(chaiHttp);
+chai.should();
+describe('Search functionality', () => {
+  it('Should search in database', (done) => {
+    chai.request(app)
```

Comment:
`Should search in database` appears in multiple places. Kindly use more descriptive names for the test cases.